code/modules/reagents/chemistry/recipes/pyrotechnics.dm

```diff
@@ -129,10 +129,11 @@
 	mix_sound = null
 
 /datum/chemical_reaction/blackpowder_explosion/on_reaction(datum/reagents/holder, created_volume)
-	var/location = get_turf(holder.my_atom)
+	var/turf/location = get_turf(holder.my_atom)
 	var/datum/effect_system/spark_spread/s = new /datum/effect_system/spark_spread
 	s.set_up(2, 1, location)
 	s.start()
+	message_admins("Black Powder explosion imminment at [location.x] [location.y] [location.z] [ADMIN_JMP(location)] with volume [created_volume]")
```

Comment:
This is pretty redundant. We already get notification of when it actually does explode. This is just going to spam them with twice the logs.

They're not going to have enough time to stop it, let alone react to it, in time, anyway.

Reply:
True it doesn't really solve the issue I set out to solve.

Reply:
Remove this, please.

Reply:
Removed